helm/cluster-openstack/templates/_openstack_cluster_template.tpl

```diff
@@ -43,6 +48,14 @@ spec:
         instance:
           flavor: {{ .Values.bastion.flavor }}
           image: {{ .Values.bastion.image }}
+          {{- if not .Values.nodeCIDR }}
+          networks:
+          - filter:
+              name: {{ .Values.networkName }}
+            subnets:
+            - filter:
+                name: {{ .Values.subnetName }} 
+          {{- end }}
```

Comment:
Having all bastions in the same network is strange. But on the other hand they they all can be accessed with the same signed key. Something to keep in mind. Maybe we'll need to address that at some point.